assets/src/edit-story/elements/text/icon.js

```diff
@@ -18,15 +18,25 @@
  * External dependencies
  */
 import { __ } from '@web-stories-wp/i18n';
-
+import styled from 'styled-components';
 /**
  * Internal dependencies
  */
-import { ReactComponent as Icon } from './icon.svg';
+import { Icons } from '../../../design-system';
+
+const IconContainer = styled.div`
+  height: auto;
+  width: 30px;
```

Comment:
Isn't it 32px?

![Screen Shot 2021-03-02 at 21 09 29](https://user-images.githubusercontent.com/637548/109741710-9d0db280-7b9b-11eb-82e8-d40f9ae6da50.png)